tests/testthat/_snaps/bin2factor.md

```diff
@@ -14,15 +14,17 @@
       rec %>% step_bin2factor(rocks, levels = letters[1:5])
     Condition
       Error in `step_bin2factor()`:
-      ! `levels` should be a two element character string
+      x `levels` should be a two element character string.
+      i 5 elements were supplied.
 
 ---
 
     Code
       rec %>% step_bin2factor(rocks, levels = 1:2)
     Condition
       Error in `step_bin2factor()`:
-      ! `levels` should be a two element character string
+      x `levels` should be a two element character string.
+      i It was a an integer vector.
```

Comment:
```suggestion
      i It was an integer vector.
```

I understand a/an is a hard problem technically, but we ought not to duplicate both articles.

Reply:
done in https://github.com/tidymodels/recipes/pull/1255/commits/483af8a84507657d909bec152f8bb87e15fb9350